src/index.js

```diff
@@ -1,32 +1,47 @@
 var XMLHttpRequest = require("xmlhttprequest").XMLHttpRequest;
```

Comment:
En el problema 2 debíamos utilizar ECMAScript 6 por lo tanto ya no utilizamos 'var'

```suggestion
const XMLHttpRequest = require("xmlhttprequest").XMLHttpRequest;
```

Documentación:
https://2017-web-development.readthedocs.io/en/latest/final/lana/